content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -139,4 +134,31 @@ table](https://github.com/kubernetes-sigs/cri-o/blob/master/docs/crio.conf.5.md#
 See cri-o's config documentation for more details:
 https://github.com/kubernetes-sigs/cri-o/blob/master/cmd/crio/config.go
 
+
```

Comment:
nit: above in line 109, under `dockershim`:
Kubernete's built-in dockershim CRI does not support runtime handlers.
Change to Kubernetes